frontend/package.json

```diff
@@ -25,7 +25,7 @@
     "@types/node": "^16.18.36",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.8",
-    "amazon-quicksight-embedding-sdk": "^2.2.2",
+    "amazon-quicksight-embedding-sdk": "2.3.1",
```

Comment:
Have you reported the issue to service team?

Reply:
Yes.